karma.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: karma
+  version: "0.120"
+  description: "A dashboard for managing alerts from Alertmanager"
+  epoch: 0
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prymitive/karma.git
+      tag: v${{package.version}}
+      expected-commit: 8009f4aa581980654e5c67b26b41e85d9e7ccce0
+
+  - name: install-npm-dependencies
+    working-directory: ui
+    pipeline:
+      - runs: |
+          mkdir -p node_modules/.bin
+          touch node_modules/.install
+          npm install --package-lock-only
+          npm ci
```

Comment:
```suggestion
          npm ci --omit=dev
```

Reply:
change implemented

Reply:
The build fails when `npm ci` is run for only production either by adding `--omit=dev` or `--production`

Reply:
Its weird, what was the issue? We're just omitting the dev packages that don't need to be exist on the runtime.